scripts.js

```diff
@@ -1,68 +1,82 @@
-var apiurl, myresult;
-
-headerurl = "https://api.flickr.com/services/rest/?method=flickr.photosets.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&photoset_id=72157655762735509&privacy_filter=1&per_page=1&format=json&nojsoncallback=1";
-
-apiurl =
-  "https://api.flickr.com/services/rest/?method=flickr.galleries.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&gallery_id=72157690043876864&per_page=8&format=json&nojsoncallback=1";
-
+var myResult;
+var headerUrl =
+  'https://api.flickr.com/services/rest/?method=flickr.photosets.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&photoset_id=72157655762735509&privacy_filter=1&per_page=1&format=json&nojsoncallback=1';
+var apiUrl =
+  'https://api.flickr.com/services/rest/?method=flickr.galleries.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&gallery_id=72157690043876864&per_page=8&format=json&nojsoncallback=1';
 
+$(window).on('load', function() {
+  AOS.init({
+    duration: 1200
+  });
 
-$(window).on('load', function () {	
-	AOS.init({
-  		duration: 1200,
-	})
+  $.getJSON(headerUrl, function(json) {
+    $.each(json.photoset.photo, function(i, myResult) {
+      var imgSrc =
+        'http://farm' +
+        myResult.farm +
+        '.static.flickr.com/' +
+        myResult.server +
+        '/' +
+        myResult.id +
+        '_' +
+        myResult.secret +
+        '_b.jpg';
+      var imgThumb = $('<img>').attr({ src: imgSrc, id: 'header-img' });
+      $(imgThumb).appendTo('.main');
+    });
+  });
 
-	$.getJSON(headerurl, function(json) {
-      $.each(json.photoset.photo, function(i, myresult) {
-      	var img_src = "http://farm" + myresult.farm + ".static.flickr.com/" + myresult.server + "/" + myresult.id + "_" + myresult.secret + "_b.jpg";
-      	var img_thumb = $("<img>").attr({"src":img_src, "id":"header-img"})
-      	$(img_thumb).appendTo(".main");
+  $.getJSON(apiUrl, function(json) {
+    $.each(json.photos.photo, function(i, myResult) {
+      var imgSrc =
+        'http://farm' +
+        myResult.farm +
+        '.static.flickr.com/' +
+        myResult.server +
+        '/' +
+        myResult.id +
+        '_' +
+        myResult.secret +
+        '_b.jpg';
+      var imgThumb = $('<img>').attr({
+        src: imgSrc,
+        id: 'img-id',
+        class: 'hvrbox-layer_bottom'
```

Comment:
Add a class for your images to change the declaration in your CSS for `#img-id:hover`.